airbyte-webapp/package.json

```diff
@@ -154,7 +152,7 @@
     "orval": "^6.11.0-alpha.10",
     "prettier": "^2.6.2",
     "react-select-event": "^5.5.0",
-    "storybook-addon-mock": "^2.4.1",
```

Comment:
:information_source: This isn't compatible with Storybook 7 anymore. I've checked the only place it was used `ConnectorForm` and it seems this endpoint is no longer called during storybook (I assume this was resolved due to some refactoring we did in this form), so we no longer need to mock any data in storybook.